### ISSUE TITLE
UML for Pen interfaces

### ISSUE THREAD

**Issue by Catyell2** [Catyell2/POJO#8]
![pen w_ interfaces - page 1](https://cloud.githubusercontent.com/assets/25180103/24810424/05739948-1b88-11e7-95ac-3bab9d7ae760.jpeg)

**Comment by Catyell2** [comment]
@anthonykulis this is the rough UML for Pen interfaces.

**Comment by anthonykulis** [comment]
nah. interfaces are "actions". `Write` and `Clickable` are ok. `Capable` is not textually correct. Change that to `Coverable` maybe? Or maybe `Attachable` and `Removable` and you attach a cap and remove a cap? Change `Write` to the adjective `Writable`. `ColorChoice` is not an action. It is a property. Because some pens are refillable, I'd suggest that interface instead as `Fillable`. Also, if you are using lucid charts, there is a interface model. Just add more shapes and select UML. Its on the top row to the right. Just keep `<<interface>>` and change `Interface` in that module to your class. Also, use the class model for classes. Its the first one in the list (top left). Then get this code done. You do not need ot get it anymore approved by me if you follow my instructions.

**Comment by Catyell2** [comment]
![pen w_ interfaces - page 1 1](https://cloud.githubusercontent.com/assets/25180103/24812381/715db9d4-1b8f-11e7-8fb0-b3bf1586beae.jpeg)

I didn't have the UML options before, fixed that.
